src/network/macvlan_dhcp.rs

```diff
@@ -6,6 +6,9 @@ use std::str::FromStr;
 
 use crate::dhcp_proxy::lib::g_rpc::NetworkConfig;
 use crate::dhcp_proxy::proxy_conf::DEFAULT_UDS_PATH;
+
+pub type DhcpLeaseInfo = (Vec<NetAddress>, Option<Vec<IpAddr>>, Option<Vec<String>>);
```

Comment:
it may be better to define a struct so that the fields have names so it is easier for the reader to understand

But this is just a minor nit not blocking and I am fine to merge it like this.